doc/api/test.md

```diff
@@ -920,6 +920,61 @@ test('runs timers as setTime passes ticks', (context) => {
 });
 ```
 
+## Snapshot testing
+
+> Stability: 1.0 - Early development
+
+Snapshot tests allow arbitrary values to be serialized into string values and
+compared against a set of known good values. The known good values are known as
+snapshots, and are stored in a snapshot file. Snapshot files are managed by the
+test runner, but are designed to be human readable to aid in debugging. Snapshot
+files should be checked into source control along with your test files. In order
```

Comment:
```suggestion
test runner, but are designed to be human readable to aid in debugging. Best practice is for snapshot
files to be checked into source control along with your test files. In order
```
I think better to recommend this instead of saying it "should be"